pootle/static/js/browser/components/Stats.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) Pootle contributors.
+ *
+ * This file is a part of the Pootle project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import StatsAPI from 'api/StatsAPI';
+import TopContributors from './TopContributors'
+
+
+const Stats = React.createClass({
+
+  propTypes: {
+    topContributors: React.PropTypes.array.isRequired,
+    allTopContributorsLoaded: React.PropTypes.bool.isRequired,
+    pootlePath: React.PropTypes.string.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      topContributors: this.props.topContributors,
+      allTopContributorsLoaded: this.props.allTopContributorsLoaded,
+    };
+  },
+
+  loadMoreTopContributors() {
+    if (!this.state.allTopContributorsLoaded) {
+      const params = {offset: this.state.topContributors.length};
+      return StatsAPI['getTopContributors'](this.props.pootlePath, params)
+        .done(this.onLoadMoreTopContributors);
+    }
+  },
+
+  onLoadMoreTopContributors(data) {
+    const topContributors = this.state.topContributors.concat(data.items);
+    this.setState({
+      topContributors: topContributors,
+      allTopContributorsLoaded: data.all_items_loaded
```

Comment:
@julen I'm glad that you raised this question. Then we need to add one more not very fast query to get total count. So I tried to get one extra unit to understand if we can load more. Though I believe we will need to cache top contributors stats some how in future.